### PR TITLE
CI: fix

### PR DESCRIPTION
**Pull request by gabsuren**
-  fix of websocket host example
- CI: allow unsecure node version for self hosted runner

### CODE REVIEW

**Comment by gabsuren** [comment]
Note: Added other labels to test full CI status

File: components/esp_websocket_client/examples/linux/main/CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,10 @@ idf_component_register(SRCS "main.c"
                     "."
                     REQUIRES esp_websocket_client protocol_examples_common)
 
+idf_component_get_property(log_lib log COMPONENT_LIB)
+
+target_compile_options(${log_lib} PRIVATE -Wno-implicit-function-declaration)
```
Comment (euripedesrocha) [inline components/esp_websocket_client/examples/linux/main/CMakeLists.txt:8]: Could you add the original context for this fix?

File: components/esp_websocket_client/examples/linux/CMakeLists.txt [hunk 2]
```diff
@@ -6,10 +6,12 @@ set(common_component_dir ../../../../common_components)
...
-  "${common_component_dir}/linux_compat"
   "${common_component_dir}/linux_compat/freertos"
    $ENV{IDF_PATH}/examples/protocols/linux_stubs/esp_stubs
    $ENV{IDF_PATH}/examples/common_components/protocol_examples_common)
 
 set(COMPONENTS main)
 project(websocket)
+
+idf_component_get_property(websocket esp_websocket_client COMPONENT_LIB)
+target_compile_definitions(${websocket} PRIVATE "-DCONFIG_IDF_TARGET_LINUX")
```
Comment (david-cermak) [inline components/esp_websocket_client/examples/linux/CMakeLists.txt:17]: wouldn't it be easier to `#include "sdkconfig.h"` in the relevant modules
(if they uses those `CONFIG_...` defines, they should include it)

**Review by david-cermak — APPROVED** [review]
LGTM!

**Pull request merged**